packages/atomic/src/components/product-listing/atomic-product-listing-pager/atomic-product-listing-pager.tsx

```diff
@@ -0,0 +1,127 @@
+import {SearchStatus, SearchStatusState} from '@coveo/headless';
```

Comment:
This a problematic import: 

By mixing imports from both `@coveo/headless` and `@coveo/headless/product-listing` just below, this defeats the purpose of having different sub packages to have an optimized payload that contains only the necessary code needed to power product listing.

With this import it means you are pulling in a bunch of unrelated code to product listing as part of the final shipped bundle to end users.


The correct approach will be to mimic the job that has been done for `headless-insight-status` (ie: create a `headless-product-listing-status` and re-export that new function from `product-listing.index.ts`

Reply:
Will do, thanks for the explanation.